esmvaltool/diag_scripts/seaice/seaice_tsline.ncl

```diff
@@ -0,0 +1,327 @@
+  else  ; select a month
+    val_area&time@units = smonths(stringtoint(diag_script_info@month) - 1)
+  end if
+  val_area@units = "10~S~6~N~ km~S~2~N~"
+  val_area@long_name = "Sea Ice Area"
+
+  ; *********** PLOTTING ************
+  if (dimsizes(val_ext&time) .eq. 1) then
+    log_info(" Skipping tsline: " + \
+             "plot needs at least two time values, has 1")
+  else
+    ; Create plot variables
+    wks_ext = get_wks(0, DIAG_SCRIPT, outfile_ext)
+    wks_area = get_wks(0, DIAG_SCRIPT, outfile_area)
+
+    ; Define plot resources
+    res                 = True
+    res@tmXBLabelAngleF = 45
+    res@tmXBLabelJust   = "CenterRight"
+
+    ; Min and max values on x-axis, leave space on right for legend
+    res@trXMaxF = max(val_ext&time) + 0.25 * (max(val_ext&time) - \
+                                              min(val_ext&time))
+    res@trXMinF = min(val_ext&time) - 0.05 * (max(val_ext&time) - \
+                                              min(val_ext&time))
+    res@tmXBMode = "Automatic"
+    res@tmXBFormat = "f"
+    if (isatt(diag_script_info, "xy_line_legend")) then
+      if (.not.diag_script_info@xy_line_legend) then
+        res@trXMaxF = max(val_ext&time)
+        res@trXMinF = min(val_ext&time)
+      end if
+    end if
+
+    ; Resources specific for extent plot
+    res_ext = True
+    copy_VarMeta(res, res_ext)
+    res_ext@tiMainString = val_ext&time@units + " " + \
+      diag_script_info@region + " " + val_ext@long_name
+    res_ext@tiYAxisString = val_ext@units
+
+    ; Resources specific for area plot
+    res_area = True
+    copy_VarMeta(res, res_area)
+    res_area@tiMainString  = val_area&time@units + " " + \
+      diag_script_info@region + " " + val_area@long_name
+    res_area@tiYAxisString = val_area@units
+
+    ; Draw the plots (procedure in ./plot_scripts/ncl/xy_line.ncl)
+    xy_line(wks_ext, val_ext, val_ext&time, val_ext_stddev, res_ext, info)
+    if (isatt(diag_script_info, "legend_outside")) then
+      ; Legend can only once be drawn in an extra file (hlu error),
+      ; but is same for both area and extent anyway
+      if (diag_script_info@legend_outside) then
+        val_area@legend_outside = True
+      end if
+    else
+      diag_script_info@legend_outside = False
+    end if
+    xy_line(wks_area, val_area, val_area&time, val_area_stddev, res_area, info)
+
+    ; Provenance
+    statistics = (/"mean", "stddev", "clim"/)
+    domain = diag_script_info@region
+    plottype = "tsline"
+    authors = (/"senf_da", "gott_kl"/)
+    references = "stroeve07grl"
+    infiles = metadata_att_as_array(info, "filename")
+    val_ext@var = var0
+    val_ext@diag_script = DIAG_SCRIPT
+    if (isatt(wks_ext, "fullname")) then
+      outfile_ext = wks_ext@fullname
+    else
+      outfile_ext = outfile_ext + "." + output_type()
+    end if
+    caption_ext = "Timeseries of " + diag_script_info@month + " " + \
+      diag_script_info@region + " sea ice extent in " + val_ext@units
```

Comment:
Please include `smonths(toint(diag_script_info@month) - 1)` in the caption for the name of the month.